src/main/scala/co/ledger/wallet/daemon/controllers/AccountsController.scala

```diff
@@ -114,6 +114,12 @@ class AccountsController @Inject()(accountsService: AccountsService) extends Con
         accountsService.getBalance(request.contract, request.accountInfo)
       }
 
+      // End point queries for account xpub
+      get("/xpub") { request: XpubRequest =>
```

Comment:
Don't have to add a new request type `XpubRequest`, use `AccountRequest` instead
```suggestion
      get("/xpub") { request: AccountRequest =>
```